runconfig/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 	Hostname        string                // Hostname
 	Domainname      string                // Domainname
 	User            string                // User that will run the command(s) inside the container
+	ArgsEscaped     bool                  `json:",omitempty"` // True if command is already escaped
```

Comment:
On second thought, I think it does need to live in Config, since it is not a runtime configuration, but rather reflecting the state of the Command.